themes/socialblue/templates/group/group--statistic--sky.html.twig

```diff
@@ -0,0 +1,40 @@
+<div class="card__counter">
+  <ul>
+    <li>
+      <span class="card__counter-quantity">{{ group_events }}</span>
+      <span class="card__counter-text">{% trans %}event{% plural group_events %}events{% endtrans %}</span>
+    </li>
+    <li>
+      <span class="card__counter-quantity">{{ group_topics }}</span>
+      <span class="card__counter-text">{% trans %}topic{% plural group_topics %}topics{% endtrans %}</span>
+    </li>
+    <li>
+      <span class="card__counter-quantity">{{ group_members }}</span>
+      <span class="card__counter-text">{% trans %}member{% plural group_members %}members{% endtrans %}</span>
+    </li>
+  </ul>
```

Comment:
Can we in some way make this more generic or fieldable? It think we're using this structure more often going forward and we could set this up in such a way that we don't need to adjust a template when we want to add a statistic (or have an extension add a statistic or change what's shown).